.github/workflows/server-lint-and-test.yaml

```diff
@@ -29,15 +29,15 @@ jobs:
     steps:
       - name: Checkout Code
         uses: actions/checkout@v5
+        with:
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+          ref: ${{ github.event.pull_request.head.sha }}
 
       - name: Set up Helm
         uses: azure/setup-helm@v4
 
       - name: Set up chart-testing
         uses: helm/chart-testing-action@v2.7.0
-        with:
-          repository: ${{ github.event.pull_request.head.repo.full_name }}
-          ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
This was not a valid place for this config - should be in the `checkout` action step